tests/acceptance/00_basics/06_host_specific_data/01-vars.cf.sub

```diff
@@ -0,0 +1,6 @@
+bundle agent __main__ {
+  reports:
+      "$(data:variables.test_variable)";
+      "$(with)" with => format("%S", "data:scope1.test_variable");
```

Comment:
cool. I forgot about `format`, I've been using `storejson` a lot recently with `with` for reports. :+1: